src/labels/CustomLabels.labels

```diff
@@ -4214,6 +4270,548 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Alt text: Arrow button to move section up</shortDescription>
+        <!-- {0} is a string. Name of a form section e.g. Gift Entry Form -->
+        <value>Move the {0} section up.</value>
+    </labels>
+    <labels>
+        <fullName>geAssistiveSpinner</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Alt text: lightning-spinner</shortDescription>
+        <value>Content is loading.</value>
+    </labels>
+    <labels>
+        <fullName>geBodyBatchHeaderLeftCol</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Text block about adding or removing fields in the batch header</shortDescription>
+        <value>Choose fields to include in your Batch Header. Select the checkbox to add a field and deselect to remove the field from the form. If you'd like additional custom fields available for the Batch Header, add them in the NPSP Data Import Batch object.</value>
+    </labels>
+    <labels>
+        <fullName>geBodyBatchHeaderRightCol</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Text block about setting up batch header fields</shortDescription>
+        <value>For each field, you can edit the field label, mark it as required, or assign a default value. Default values can be overwritten by the person who creates the batch.</value>
+    </labels>
+    <labels>
+        <fullName>geBodyBatchHeaderWarning</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Warning body of required field added to Batch Header</shortDescription>
+        <!-- {0} is comma separated list of field api names from the Data Import Batch object -->
+        <!-- e.g. npsp__GiftBatch__c, npsp__Batch_Description__c -->
+        <value>These required fields were automatically added to the Batch Header:</value>
+    </labels>
+    <labels>
+        <fullName>geBodyFormFieldsLeftCol</fullName>
+        <categories>Gift Entry</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Template Builder Form Fields tab left column body</shortDescription>
+        <value>Choose fields to include in your form. Before adding fields, click the section in the main screen where you want the fields to go. In the sidebar, select the checkbox to add a field, and deselect it to remove the field from the form. If the object or field you want to use isn't listed, make sure you've set up the field with Advanced Mapping.</value>
+    </labels>
+    <labels>
+        <fullName>geBodyFormFieldsLeftColAdditional</fullName>
```

Comment:
@psadie-sfdo the font size of this additional info appears larger than the primary text. Is that intentional?

Reply:
@melissabarber - Good catch! Not intentional and updating this.